parser.go

```diff
@@ -1367,9 +1367,7 @@ func (p *parser) parseSchemaObject(pkgPath, pkgName, typeName string, register b
 		if err != nil {
 			return nil, err
 		}
-		schemaObject.Type = newSchema.Type
-		schemaObject.Properties = newSchema.Properties
-		schemaObject.AdditionalProperties = newSchema.AdditionalProperties
```

Comment:
Do we no longer need to set these other properties? Don't understand why these lines are deleted.

Reply:
No, because in that case it should be a Ref to the correct struct.